RecoParticleFlow/PFProducer/python/particleFlow_cfi.py

```diff
@@ -70,12 +70,24 @@
     maxEeleOverPout = cms.double(0.2),
     maxDPhiIN = cms.double(0.1)    
     ),
+    electron_protectionsForBadHcal = cms.PSet(
+        enableProtections = cms.bool(False),   
+        full5x5_sigmaIetaIeta = cms.vdouble(0.0106, 0.0387), # EB, EE; 94Xv2 cut-based medium id
+        eInvPInv = cms.vdouble(0.184, 0.0721),
+        dEta = cms.vdouble(0.0032*2, 0.00632*2), # relax factor 2 to be safer against misalignment
```

Comment:
this sounds as if 0.0032 and 0.00632 are known values from elsewhere.
Perhaps the rest are like that as well.
Is it possible to reuse those constants instead of having copies here?

Reply:
numbers come from https://github.com/cms-sw/cmssw/blob/master/RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_Fall17_94X_V2_cff.py
however, I don't know if we would want to introduce a dependency on this file